WordPress/src/test/java/org/wordpress/android/ui/posts/EditPostPublishSettingsJetpackSocialUiStateMapperTest.kt

```diff
@@ -23,110 +25,148 @@ class EditPostPublishSettingsJetpackSocialUiStateMapperTest {
         localeProvider = localProvider,
     )
 
+//    private val postSocialSharingModel = PostSocialSharingModel(
```

Comment:
These tests will be updated in my next PR.

Reply:
Fixed [here](https://github.com/wordpress-mobile/WordPress-Android/pull/18853)